src/openms/source/ANALYSIS/QUANTITATION/PeptideAndProteinQuant.cpp

```diff
@@ -893,6 +889,7 @@ namespace OpenMS
         for (auto const & s : total_abundances)
         {
           // Note: sample indices are one-based
+          // WARNING: THOSE LINES ASSUME THAT THE EXPERIMENTAL DESIGN
```

Comment:
what?

Reply:
Still trying to figure it out if this is a problem for more general designs

Reply:
Hmmm I am not sure right now why a -1 is needed here.
Sample indices should be zero-based now.

Reply:
Are there tests for this?